addon/components/paper-item.js

```diff
@@ -10,100 +23,105 @@ export default Ember.Component.extend(RippleMixin, ProxyMixin, {
   center: false,
   dimBackground: true,
   outline: false,
-  noink: Ember.computed.not('shouldBeClickable'),
 
   classNameBindings: ['shouldBeClickable:md-clickable', 'hasProxiedComponent:md-proxy-focus'],
   attributeBindings: ['role', 'tabindex'],
   role: 'listitem',
   tabindex: '-1',
 
-  hasProxiedComponent: Ember.computed.bool('proxiedComponents.length'),
+  hasProxiedComponent: bool('proxiedComponents.length'),
 
-  hasSecondaryAction: Ember.computed('secondaryItem', 'onClick', function() {
-    let secondaryItem = this.get('secondaryItem');
-    return secondaryItem && (secondaryItem.action || (this.get('onClick') && this.isProxiedComponent(secondaryItem)));
-  }),
+  hasPrimaryAction: notEmpty('onClick'),
 
-  secondaryItem: Ember.computed('proxiedComponents.[]', function() {
-    let proxiedComponents = this.get('proxiedComponents');
-    return proxiedComponents.find(function(component) {
-      return component.classNames.indexOf('md-secondary') !== -1;
-    });
+  hasSecondaryAction: computed('secondaryItem', 'onClick', {
+    get() {
```

Comment:
We're not using the `get`/`set` syntax. Please use the normal `function() {` for getter computeds.